spec/system/purchase_system_spec.rb

```diff
@@ -24,6 +26,17 @@
       page.refresh
       expect(page).to have_text("Purchased Date")
       expect(page).to have_text(1.week.ago.strftime("%Y-%m-%d"))
+      expect(page).to have_text(@total_value_all_purchases)
```

Comment:
Can you confirm what the value to rspec of `@total_value_all_purchases` is? I wonder if this is just checking that it has text of nil which is always true.

Reply:
Nice catch! `@total_value_all_purchases` is `nil` here.
However, I will just delete this line, because confirming its value is already done on the below spec "User sees total purchases value".

Reply:
Sounds good!